perun-openapi/openapi.yml

```diff
@@ -566,6 +566,7 @@ components:
       allOf:
         - $ref: '#/components/schemas/Auditable'
         - properties:
+            blocked: {type: boolean}
```

Comment:
Shouldn't this be a part of ``RichDestination`` instead? I don't see such property or method in ``Destination`` object in perun core.

Reply:
It is in RichDestination, see https://github.com/CESNET/perun/blob/master/perun-base/src/main/java/cz/metacentrum/perun/core/api/RichDestination.java#L55